integration-tests/docker/environment-configs/common

```diff
@@ -72,6 +72,7 @@ druid_coordinator_kill_supervisor_on=true
 druid_coordinator_kill_supervisor_period=PT10S
 druid_coordinator_kill_supervisor_durationToRetain=PT0M
 druid_coordinator_period_metadataStoreManagementPeriod=PT10S
+druid_sql_planner_authorizeSystemTablesDirectly=true
```

Comment:
Seems like it might be wasteful to have integration tests for both true and false, but since the default is disabled here, I think we may still want to have ITs for when this feature is disabled.

Reply:
I considered this, I think it is ok to only test enabled because it just layers additional checks on top of the disabled code path, so with the newly added user we should still be covering all of the code paths that were previously tested, just distributed a bit differently.